### PR TITLE
Implement new UI for edit page node

### PR DESCRIPTION
**Pull request by jomunker**
This PR implements the new design for the Edit Page Node Dialog.

<details>
<summary>Screenshots Implementation</summary>
image
image
</details>

<details>
<summary>Screenshots Design</summary>
image
image
</details>

### CODE REVIEW

File: packages/admin/cms-admin/src/pages/createEditPageNode.tsx [hunk 1]
```diff
@@ -439,6 +462,9 @@ export function createEditPageNode({
                                         />
                                     )}
                                 </Field>
+
+                                <CustomDivider />
```
Comment (johnnyomair) [inline packages/admin/cms-admin/src/pages/createEditPageNode.tsx:466]: This Divider will be rendered even if there are no additionalFormFields (which come from the app). The divider should be rendered by the app I suppose

File: packages/admin/cms-admin/src/pages/createEditPageNode.tsx [hunk 2]
```diff
@@ -48,6 +49,15 @@ export interface EditPageNodeProps {
...
 }
 
+const CustomDivider = styled(Divider)`
+    margin: ${(props: { theme: Theme }) => props.theme.spacing(6)} 0;
+`;
+
+const RedirectsTooltip = styled(Tooltip)`
+    margin-left: ${(props: { theme: Theme }) => props.theme.spacing(1)};
+    padding: ${(props: { theme: Theme }) => props.theme.spacing(1)};
+`;
```
[the remark this answers is not in the document]
Comment (jomunker) [reply packages/admin/cms-admin/src/pages/createEditPageNode.tsx:59]: Then I'll remove the padding. margin-left can stay I guess or should I wrap the IconButton in a Box with margin left?

File: packages/admin/cms-admin/src/pages/createEditPageNode.tsx (hunk 1 above)
Comment (jomunker) [reply packages/admin/cms-admin/src/pages/createEditPageNode.tsx:466]: So should I only render it if additional form fields are present or how should I proceed with this?

File: packages/admin/cms-admin/src/pages/createEditPageNode.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (johnnyomair) [reply packages/admin/cms-admin/src/pages/createEditPageNode.tsx:59]: Wrap the IconButton 👍🏼

File: packages/admin/cms-admin/src/pages/createEditPageNode.tsx (hunk 1 above)
Comment (jomunker) [reply packages/admin/cms-admin/src/pages/createEditPageNode.tsx:466]: Alright thanks 👍🏻 will do

File: demo/admin/src/common/EditPageNode.tsx [hunk 3]
```diff
@@ -38,8 +38,16 @@ export const EditPageNode = createEditPageNode({
     },
     nodeFragment: additionalPageTreeNodeFieldsFragment,
     additionalFormFields: (
-        <div>
-            <Field label={<FormattedMessage id="pageTreeNode.fields.userGroup" defaultMessage="User-Group" />} name="userGroup" fullWidth>
+        <>
+            <Box marginY={6}>
+                <Divider />
+            </Box>
```
Comment (johnnyomair) [inline demo/admin/src/common/EditPageNode.tsx:44]: You could also do `<Divider sx={{ marginY: 6 }}/>`. But this is fine.

**Review by johnnyomair — APPROVED** [review]

File: demo/admin/src/common/EditPageNode.tsx (hunk 3 above)
Comment (jomunker) [reply demo/admin/src/common/EditPageNode.tsx:44]: Ah okay. What does 'sx' stand for?

Comment (johnnyomair) [reply demo/admin/src/common/EditPageNode.tsx:44]: I don't know 😅 It's a MUI feature: https://mui.com/system/getting-started/the-sx-prop/

Comment (jomunker) [reply demo/admin/src/common/EditPageNode.tsx:44]: Ah okay interesting^^ Thanks 🙏🏻

**Pull request merged**